cmd/manager/main.go

```diff
@@ -77,6 +80,7 @@ func main() {
 	flag.StringVar(&unpackImage, "unpack-image", "quay.io/operator-framework/rukpak:v0.12.0", "The unpack image to use when unpacking catalog images")
 	flag.StringVar(&systemNamespace, "system-namespace", "", "The namespace catalogd uses for internal state, configuration, and workloads")
 	flag.StringVar(&storageDir, "catalogs-storage-dir", "/var/cache/catalogs", "The directory in the filesystem where unpacked catalog content will be stored and served from")
+	flag.StringVar(&catalogServerAddr, "catalogs-server-addr", "127.0.0.1:8083", "The address where the unpacked catalogs' content will be accessible")
```

Comment:
Nit:  do we need the full 127.0.0.1 here?
```suggestion
	flag.StringVar(&catalogServerAddr, "catalogs-server-addr", ":8083", "The address where the unpacked catalogs' content will be accessible")
```

Reply:
This is saying "serve at localhost:8083 by default". The flag passed to the binary (with value ":8083") takes care of making sure the port is reachable from other addressed (I.e <service-name>.<service-namespace>.svc:8083).

Isn't totally required for anything that's being done in this PR, just best practice.